ts/features/payments/checkout/hooks/usePaymentFailureSupportModal.tsx

```diff
@@ -82,8 +89,12 @@ const usePaymentFailureSupportModal = ({
     "";
 
   const zendeskAssistanceLogAndStart = () => {
+    const { value, zendeskCategoryId } = getSubCategoryFromFaultCode(
+      zendeskPaymentCategory,
+      faultCodeDetail
+    );
     resetCustomFields();
-    addTicketCustomField(zendeskCategoryId, zendeskPaymentCategory.value);
+    addTicketCustomField(zendeskCategoryId, value);
```

Comment:
In this case, shouldn’t the `categoryId` always be present and set to the `categoryId` of the "pagamento pagoPA"? Then, the dynamic part would be the subcategory. Wouldn’t it be necessary to associate both the `categoryId` (which in this case is a static id) and the `subcategoryId` with a ticket?